backend/services/prisonerProfileService.js

```diff
@@ -16,21 +21,39 @@ module.exports = (elite2Api, keyworkerApi) => {
       inactiveAlertCount,
     } = prisonerDetails
 
-    const prisonersActiveAlertCodes = alerts.filter(alert => !alert.expired).map(alert => alert.alertCode)
+    const [iepDetails, keyworkerSessions, keyworkerDetails, userRoles, userCaseloads] = await Promise.all([
+      elite2Api.getIepSummary(context, [bookingId]),
+      elite2Api.getCaseNoteSummaryByTypes(context, { type: 'KA', subType: 'KS', numMonths: 1, bookingId }),
+      keyworkerApi.getKeyworkerByCaseloadAndOffenderNo(context, agencyId, offenderNo),
+      oauthApi.userRoles(context),
+      elite2Api.userCaseLoads(context),
+    ])
 
+    const prisonersActiveAlertCodes = alerts.filter(alert => !alert.expired).map(alert => alert.alertCode)
     const alertsToShow = alertFlagValues.filter(alertFlag =>
       alertFlag.alertCodes.some(alert => prisonersActiveAlertCodes.includes(alert))
     )
 
-    const [iepDetails, keyworkerSessions, keyworkerDetails] = await Promise.all([
-      elite2Api.getIepSummary(context, [bookingId]),
-      elite2Api.getCaseNoteSummaryByTypes(context, { type: 'KA', subType: 'KS', numMonths: 1, bookingId }),
-      keyworkerApi.getKeyworkerByCaseloadAndOffenderNo(context, agencyId, offenderNo),
-    ])
+    const canViewInactivePrisoner = userRoles && userRoles.some(role => role.roleCode === 'INACTIVE_BOOKINGS')
+    const offenderInCaseload = userCaseloads.some(caseload => caseload.caseLoadId === agencyId)
+
+    const isCatToolUser = Boolean(
```

Comment:
Suspect we may need to know this elsewhere at some point but cross that bridge when we come to it, I think.